reliability_threshold: 0.7
filter_limit_lower: 0.3
filter_limit_upper: 3.0
